images/win/scripts/Installers/Install-AzureModules.ps1

```diff
@@ -3,58 +3,33 @@
 ##  Desc:  Install Azure PowerShell modules
 ################################################################################
 
+$ErrorActionPreference = "Stop"
+
+Import-Module -Name ImageHelpers -Force
```

Comment:
Why do you need explicit import this module?

Reply:
It will guarantee that Get-ToolsetContent function will be imported correctly.

Reply:
I believe it will imported automatically due to https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Azure.json#L79

Reply:
Fixed.